packages/client/src/generation/TSClient/Model.ts

```diff
@@ -41,13 +43,16 @@ import { getModelActions } from './utils/getModelActions'
 
 export class Model implements Generable {
   protected type: DMMF.OutputType
+  protected createManyAndReturnType: undefined | DMMF.OutputType
   protected mapping?: DMMF.ModelMapping
   private dmmf: DMMFHelper
   private genericsInfo: GenericArgsInfo
   constructor(protected readonly model: DMMF.Model, protected readonly context: GenerateContext) {
     this.dmmf = context.dmmf
     this.genericsInfo = context.genericArgsInfo
     this.type = this.context.dmmf.outputTypeMap.model[model.name]
+
+    this.createManyAndReturnType = this.context.dmmf.outputTypeMap.model[`CreateMany${model.name}AndReturnOutputType`]
```

Comment:
Afterthought could we pull that dmmf type name creation into it's own `getCreateManyAndReturnOutputType` helper?

Reply:
I think that's a very good idea

Reply:
Done!